### ISSUE TITLE
Add GET /workflows/{wf_id} to API

### ISSUE THREAD

**Issue by ilveroluca** [crs4/life_monitor#17]
At the moment we're missing a resource that lists the versions of a workflow that have been registered with LM.  I think we need something like

```
  /workflows/{wf_uuid}:
    get:
      parameters:
        - $ref: '#/components/parameters/wf_uuid'
     responses:
        '200':
          description: List of versions
          content:
            application/json:
              schema:
                type: array
                items: string
```

**Comment by ilveroluca** [comment]
A resource satisfying this requirement is included the current `extend-api` branch.

**Comment by kikkomep** [comment]
Obsolete. This feature is available since release v0.1.0
